initfiles/bash/etc/systemd/system/CMakeLists.txt

```diff
@@ -43,6 +44,7 @@ set(sasha "saserver" "--daemon %i" "dafilesrv.service")
 set(thor "thormaster_lcr" "--daemon %i MASTER=\${THORMASTER}:\${THORMASTERPORT}" "dafilesrv.service")
 set(thorslave "thorslave_lcr" "--daemon %i master=\${THORMASTER}:\${THORMASTERPORT} slave=.:\${SLAVEPORT} slavenum=\${SLAVENUM} logDir=${LOG_PATH}/\${THORNAME}" "dafilesrv.service")
 set(backupnode "backup.sh" "" "")
+set(toposerver "toposerver" "--logdir=${LOG_PATH}/%i --daemon %i" "dafilesrv.service")
```

Comment:
Should this be passing logdir, rather than picking it up from config files?

Reply:
I've gone ahead and removed the --logdir from systemd unit and the init_toposerver files.  Just tested and it still all works properly and is getting picked up from the settings file.